### ISSUE TITLE
Fatal error: Uncaught Error: Class 'GeoIp2\Database\Reader' not found

### ISSUE THREAD

**Issue by gaurangpadhiyar007** [to-bai/magento2-geo-store-switcher#8]
Hey,
I have got this error on server, please suggest the solution.

```
Fatal error: Uncaught Error: Class 'GeoIp2\Database\Reader' not found
app/code/Tobai/GeoIp2/Model/Database/ReaderFactory.php:43 Stack trace: #0 

```
Thanks in advance.

**Comment by ytorbyk** [comment]
Hi @gaurangpadhiyar007 
How did you install the extension? Did you install all dependences ("tobai/magento2-geo-ip2" with "geoip2/geoip2")?

**Comment by gaurangpadhiyar007** [comment]
yes, i have  install those dependencies at vendor folder. but getting same issue.
Please help to resolve it.

**Comment by gaurangpadhiyar007** [comment]
I have run command composer install instead of composer update which resolves the issue,
But extension does not seem to work.
Is there any special requirement to create an account for maxwind?

**Comment by ramsampathi** [comment]
same issue for me in magento 2.26 .please observe in below screen shot also
http://prntscr.com/lzjo3g

please give me any solution..
Thanks,

**Comment by malikabid** [comment]
@gaurangpadhiyar007 / @ramsampathi  i had the same issue when installing manually.
User composer to install the extension. It will install all dependencies in and extension in vendor folder and your issue will be resolved.

**Comment by gaurangpadhiyar007** [comment]
@malikabid Thanks, I will check and follow your instructions.
